vitrina/requests/views.py

```diff
@@ -44,3 +48,37 @@ def get_context_data(self, **kwargs):
         }
         context_data.update(extra_context_data)
         return context_data
+        
+        
+class RequestCreateView(LoginRequiredMixin, CreateView):
+    model = Request
+    form_class = RequestForm
+    template_name = 'base_form.html'
+
+    def form_valid(self, form):
+        self.object = form.save()
+        if self.request.user.is_authenticated:
+            self.object.user = self.request.user
+        return super().form_valid(form)
```

Comment:
You should not call `super()` here, because form will be saved again. You need to do:

```suggestion
        return HttpResponseRedirect(self.get_success_url())
```